666cbc39ee16
*** Reading local file: /opt/airflow/logs/dag_id=carga_de_episodios/run_id=manual__2023-05-18T15:50:24.316798+00:00/task_id=crear_tabla_episodio/attempt=1.log
[2023-05-18 15:50:25,880] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: carga_de_episodios.crear_tabla_episodio manual__2023-05-18T15:50:24.316798+00:00 [queued]>
[2023-05-18 15:50:25,898] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: carga_de_episodios.crear_tabla_episodio manual__2023-05-18T15:50:24.316798+00:00 [queued]>
[2023-05-18 15:50:25,899] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-05-18 15:50:25,900] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-05-18 15:50:25,901] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-05-18 15:50:25,926] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): crear_tabla_episodio> on 2023-05-18 15:50:24.316798+00:00
[2023-05-18 15:50:25,931] {standard_task_runner.py:52} INFO - Started process 243 to run task
[2023-05-18 15:50:25,935] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'carga_de_episodios', 'crear_tabla_episodio', 'manual__2023-05-18T15:50:24.316798+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/dag_carga_automatica_episodios.py', '--cfg-path', '/tmp/tmp0rom4jcd', '--error-file', '/tmp/tmp_ofvj1z7']
[2023-05-18 15:50:25,936] {standard_task_runner.py:80} INFO - Job 41: Subtask crear_tabla_episodio
[2023-05-18 15:50:26,017] {task_command.py:371} INFO - Running <TaskInstance: carga_de_episodios.crear_tabla_episodio manual__2023-05-18T15:50:24.316798+00:00 [running]> on host 666cbc39ee16
[2023-05-18 15:50:26,130] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=JORGE
AIRFLOW_CTX_DAG_ID=carga_de_episodios
AIRFLOW_CTX_TASK_ID=crear_tabla_episodio
AIRFLOW_CTX_EXECUTION_DATE=2023-05-18T15:50:24.316798+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-18T15:50:24.316798+00:00
[2023-05-18 15:50:26,151] {base.py:68} INFO - Using connection ID 'amazon_redshift' for task execution.
[2023-05-18 15:50:28,191] {dbapi.py:231} INFO - Running statement: 
                DROP TABLE IF EXISTS jorgeflores2311233_coderhouse.episodio;
                CREATE TABLE jorgeflores2311233_coderhouse.episodio(
                id INTEGER PRIMARY KEY,
                nombre_episodio VARCHAR(250),
                fecha_aire VARCHAR(250),
                episodio VARCHAR(250),
                fecha_creacion DATETIME
                );  
            , parameters: None
[2023-05-18 15:50:28,763] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=carga_de_episodios, task_id=crear_tabla_episodio, execution_date=20230518T155024, start_date=20230518T155025, end_date=20230518T155028
[2023-05-18 15:50:28,802] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-18 15:50:28,854] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check

